common/j-request/request.js

```diff
@@ -262,14 +220,15 @@ class Request {
                 // 这种情况下，要根据_config在全局拦截器中将其它系统的返回适配为本系统的业务对象
                 result = that.interceptor.response(result, _config)
             }
-            if (skip || result.success) { // 接口调用业务成功

```

Comment:
My aim is return the logic data from response.
e.g  
`{ code: 0, msg: "ok", data: "insert successfully!" }` just return insert successfully!

`{ code: -1, msg: "error", data: null }` will not call _config.success but _config.fail